packages/orbit-design-tokens/src/__tests__/tokensToCssVars.ts

```diff
@@ -0,0 +1,45 @@
+import transformToCssVars from "../utils/tokensToCssVars";
+
+const tokens = {
+  paletteProductNormal: "#ff0000",
+  paletteProductNormalHover: "#e50000",
+  paletteProductNormalActive: "#cc0000",
+};
+
+describe("transformToCssVars", () => {
+  it("should convert object into css variables string", () => {
+    expect(transformToCssVars({ tokens, cssClass: false })).toMatchInlineSnapshot(`
```

Comment:
we imo do not need the functionality offered when `cssClass: true`. just the list of var defs is enough